pages/common/ack.md

```diff
@@ -1,32 +1,37 @@
 # ack
 
-> A search tool like grep, optimized for programmers.
-> More information: <https://beyondgrep.com/documentation/>.
+> A search tool like grep, optimized for developers.
+> See also: `rg`, which is much faster.
+> More information: <https://beyondgrep.com/documentation>.
 
-- Find files containing "foo":
+- Search for files containing a string or regular expression in the current directory and subdirectories:
```

Comment:
Perhaps this will read better?
```suggestion
- Search for files containing a string or regular expression in the current directory recursively:
```